### PR TITLE
WIP: facets: adds barchart for year

### PR DESCRIPTION
**Pull request by ioannistsanaktsidis**
Signed-off-by: Ioannis Tsanaktsidis <ioannis.tsanaktsidis@cern.ch>

### CODE REVIEW

File: inspirehep/base/static/js/search/search_vis.js [hunk 1]
```diff
@@ -0,0 +1,246 @@
+ define(["jquery",
```
Comment (jalavik) [inline inspirehep/base/static/js/search/search_vis.js]: There are some general indentation problems in this file it seems.

File: inspirehep/base/templates/search/facets/hep.html [hunk 2]
```diff
@@ -162,6 +229,13 @@ <h4 id="filter-by-experiment" class="pointer"><i class="fa fa-chevron-down facet
   {% endif %}
   {% if 'earliest_date' in aggs and aggs['earliest_date']['buckets'] %}
     {% set sorted_year = aggs['earliest_date']['buckets']|sort_list_by_dict_val %}
+    <h4 id="filter-by-year" class="pointer"><i class="fa fa-chevron-down facet-slider"></i> Filter by Year</h4>
+    <div id="year_hist" style="padding-top: 30px;">
```
Comment (jalavik) [inline inspirehep/base/templates/search/facets/hep.html]: Inline style is discouraged. Can it be addressed via CSS selector?

File: inspirehep/base/static/js/search/search_vis.js (hunk 1 above)
Comment (ioannistsanaktsidis) [inline inspirehep/base/static/js/search/search_vis.js]: Yes, it is still in progress. After I finished I will run jsbeautifier and it will be ok ;)

Comment (jalavik) [inline inspirehep/base/static/js/search/search_vis.js]: Cool :+1:

**Comment by ioannistsanaktsidis** [comment]
All files in this PR are in the early stage. I will comment here when it is finished so as whoever wants to check. Cheerss :dancer:

**Comment by jmartinm** [comment]
To be continued in https://github.com/inspirehep/inspirehep-search-js

**Comment by ioannistsanaktsidis** [comment]
To be continued by @eamonnmag .

**Comment by jalavik** [comment]
Closing for now then.

**Pull request closed without merging**